gneiss/plot/_plot.py

```diff
@@ -387,10 +413,15 @@ def dendrogram_heatmap(output_dir: str, table: pd.DataFrame,
                  'present in this tree.')},
     parameter_descriptions={
         'metadata': ('Metadata to group the samples. '),
-        'ndim': 'Number of dimensions to highlight.'},
+        'ndim': 'Number of dimensions to highlight.',
```

Comment:
Exactly! Just using the `n` most basal splits in the command line interface.  For the python interface, you can specify what ever splits you are interested in (see [here](https://github.com/biocore/gneiss/blob/master/gneiss/plot/_heatmap.py#L28)).

The reason why we are doing this in the CLI, is because it isn't clear how to best input the splits you are interested in, along with the colors.  

tl;dr this is available in the python interface, but not the q2 interface.